src/magic.cpp

```diff
@@ -199,6 +199,8 @@ void spell_type::load( JsonObject &jo, const std::string & )
     mandatory( jo, was_loaded, "id", id );
     mandatory( jo, was_loaded, "name", name );
     mandatory( jo, was_loaded, "description", description );
+    translation default_msg( "You cast %s!" );
```

Comment:
```suggestion
    auto default_msg = translation( "You cast %s!" );
```
Needs to be surrounded with `translation()` for string extraction.